library/network_connections.py

```diff
@@ -683,7 +683,22 @@ def connection_seems_active(cls, name):
             return None
 
         if i_content.strip() != "up":
-            return False
+            # For the active bonding/bridge connection, when all the port connection
+            # profiles are brought down firstly, then the operstate of the controller
+            # device will change into "down" automatically, which denotes missing the
+            # carrier for the controller device. However, the connection for the
+            # controller device stays active in such a situation.
+            _unused_, out, _unused_ = run_env.run_command(
+                ["ip", "address", "show", content["DEVICE"]],
+                "utf-8",
+            )
```

Comment:
what exceptions can `run_command` raise? I think, any exception should be silently swallowed.

Reply:
Updated.